src/current/v24.3/data-resilience.md

```diff
@@ -0,0 +1,123 @@
+
+- HA features ensure continuous access to data without interruption even in the presence of [failures]({% link {{ page.version.version }}/demo-cockroachdb-resilience.md %}) or disruptions to maximize uptime.
+- DR tools allow for recovery from major incidents to minimize downtime and data loss.
+
+<image src="{{ 'images/v24.3/resilience.svg' | relative_url }}" alt="Diagram showing how HA features and DR tools create a resilient CockroachDB deployment." style="width:80%" />
+
+You can balance required SLAs and [recovery objectives]({% link {{ page.version.version }}/disaster-recovery-overview.md %}) with the cost and management of each of these features to build a resilient deployment.
+
+{{site.data.alerts.callout_success}}
+For a practical guide on how CockroachDB uses Raft to replicate, distribute, and rebalance data, refer to the [CockroachDB Resilience demo]({% link {{ page.version.version }}/demo-cockroachdb-resilience.md %}) demo.
+{{site.data.alerts.end}}
+
+## High availability
+
+- [**Raft replication**]({% link {{ page.version.version }}/architecture/replication-layer.md %}): CockroachDB's built-in Raft replication stores data safely and consistently on multiple nodes to ensure no downtime even during a temporary node outage. [Replication controls]({% link {{ page.version.version }}/configure-replication-zones.md %}) allow you to configure the number and location of [replicas]({% link {{ page.version.version }}/architecture/glossary.md %}#replica) to suit a deployment.
+- [**Fault tolerance**]({% link {{ page.version.version }}/demo-cockroachdb-resilience.md %}): Capabilities built in to CockroachDB to perform routine maintenance operations with minimal impact to foreground performance. For example, [online schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}), [write-ahead log failover]({% link {{ page.version.version }}/cockroach-start.md %}#write-ahead-log-wal-failover).
+- **Logical data replication (LDR)** (Preview): A cross-cluster replication tool between multiple active CockroachDB clusters, which supports a range of topologies. LDR provides eventually consistent, table-level replication.
+
+### Choose an HA strategy
+
+CockroachDB uses synchronous, built-in Raft replication to create and distribute copies of data, ensuring consistency across the data copies. The database can tolerate nodes going offline without service interruption, whether in a single-region or [multi-region]({% link {{ page.version.version }}/multiregion-overview.md %}) cluster.
+
+<table class="comparison-chart">
+  <tr>
+    <th></th>
+    <th>Single-region replication (synchronous)</th>
+    <th>Multi-region replication (synchronous)</th>
+    <th>Logical data replication (asynchronous)</th>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RPO</b>
+    </td>
+    <td>0 seconds</td>
+    <td>0 seconds</td>
+    <td>Immediate mode 0.5 seconds<br>Transactional mode ~10s of seconds</td>
```

Comment:
transactional mode is not available yet can we just say 10s of seconds and not mention the modes? this might get confusing as we also have the 'validated mode' now.